src/interface/report/PlayerLoader.js

```diff
@@ -153,8 +186,21 @@ class PlayerLoader extends React.PureComponent {
                 </Link>
               </Tooltip>
             </div>
-            <h1 style={{ lineHeight: 1.4, margin: 0 }}><Trans>Player selection</Trans></h1>
-            <small style={{ marginTop: -5 }}><Trans>Select the player you wish to analyze.</Trans></small>
+            <div className="flex wrapable" style={{ marginBottom: 15 }}>
+              <div className="flex-main" style={{ position: 'relative' }}>
```

Comment:
```suggestion
              <div className="flex-main">
```

Doesn't seem used anywhere